Src/Particle/AMReX_Particles.H

```diff
@@ -48,6 +48,10 @@
 
 #ifdef AMREX_USE_HDF5
 #include <hdf5.h>
+#ifdef AMREX_USE_HDF5_ASYNC
+#include "h5_vol_external_async_native.h"
+extern hid_t es_par_g;
```

Comment:
Could we prefix this global variable with `amrex_` or put it in `namespace amrex`?

Reply:
@WeiqunZhang  Just moved them (es_par_g and es_id_g) to amrex namespace.